lib/myads/myinterstitial_ad.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+class MyInterstitialAds {
+  InterstitialAd? _interstitialAd;
```

Comment:
see if you really need _interstitialAd object to implement the ad. It would be better if you can do it like Banner Ads